clean.py

```diff
@@ -17,7 +18,8 @@
 # Map functions: function(batch: Dict) -> Dict
 MAPS = {
     "replace_newline_with_space": replace_newline_with_space,
-    "remove_lines_with_code": remove_lines_with_code
+    "remove_lines_with_code": remove_lines_with_code,
+    "strip_substrings_kn_wikisource": build_substring_stripper([["അവലംബം","പുറത്തേക്കുള്ള കണ്ണികൾ"]]),
```

Comment:
These seem to be "Source", "External links". As mentioned the slack thread I think they belong to the document and we should only remove them if they don't occur in a natural way.